_rules/text-contrast-enhanced-09o5cg.md

```diff
@@ -164,6 +164,26 @@ This dark gray text has a contrast ratio of 12.6:1 on the white background in a
 </script>
 ```
 
+#### Passed Example 9
+
+This text is part of a widget because it is a child of a `button` element. The text has the default
+browser button text color on the default browser button background color. By default, this is black text on a
+light gray background with a contrast ratio of 18.26:1
+
+```html
+<button>My button!</button>
+```
+
+#### Passed Example 10
+
+This text is part of a widget because it is a child of an element with the `role` attribute set to `button`.
+The text has the default browser text color on the default browser background color. By default, this is 
+black text on a white background with a contrast ratio of 21:1
+
```

Comment:
Same comments on the examples, I don't think that "part of a widget" is needed anymore.